stellarphot/photometry/tests/test_profiles.py

```diff
@@ -206,7 +206,7 @@ def test_radial_profile_with_background(self):
             np.testing.assert_allclose(
                 rad_prof.curve_of_growth.profile[-1],
                 expected_integral,
-                atol=1.5 * expected_stddev,
+                atol=2.0 * expected_stddev,
```

Comment:
Does the comment for this function need to be adjusted as well since we are now at 2.0 standard deviations? Also did the number of standard deviations need to be adjusted due to the new seed?

Reply:
Ah, yes. Fixed that.

Reply:
Good catch!